### DISCUSSION TITLE
post migration to sentry-ruby/rails, no new issues are seen - any way to read verbose logs?

### DISCUSSION THREAD

**Discussion by jywarren** [getsentry/support#20] [Support]
I recently tried updating our app to `sentry-ruby` and `sentry-rails` in https://github.com/publiclab/mapknitter/pull/1712#issuecomment-1154844893, and although it's very similar to another such migration I recently did (linked from that comment), I'm not seeing any issues appearing in Sentry. I have even manually triggered a message from the console and it did not appear:

```
irb(main):007:0> ENV['SENTRY_DSN']
=> "https://d51bdd76485d488c8104efdbcb109ba2@o239675.ingest.sentry.io/1726938"  
irb(main):006:0> Sentry.capture_message("tests", extra: { debug: true })
=> nil
```

I've checked the settings pretty closely and haven't found any that seem out of place. 

Is there something else I should check that I might have missed? Thank you!

For convenience here is my config (edited for clarity):

```ruby
 Sentry.init do |config| 
   config.environment = ENV["COMPOSE_PROJECT_NAME"] || ENV["RAILS_ENV"] || %w(production) 
   config.enabled_environments = %w[production, mapknitter_stable, mapknitter_unstable] 
   config.breadcrumbs_logger = [:sentry_logger, :http_logger] 
   config.traces_sample_rate = 0.5 

   # left this as it's not clear if it's needed anymore:
   filter = ActionDispatch::Http::ParameterFilter.new(Rails.application.config.filter_parameters) 
   config.before_send = lambda do |event, hint| 
     filter.filter(event.to_hash) 
   end 
 end 
```

****

If it's helpful, i was able to install the JavaScript tag and send `Sentry.captureMessage('hello')` and generated an issue that way: https://sentry.io/share/issue/9ab0dd17e5c74904973d2693bd47f0c0/

But in Ruby, no such luck. 

****

**Update:** For what it's worth, i got the same lack of issue when i tried creating a message from the console in our plots2 repository, where sentry is working fine: 
```
[2] pry(main)> ENV['SENTRY_DSN']
=> "https://0490297edae647b3bd935bdb4658da54@o239675.ingest.sentry.io/1410626"
[3] pry(main)> Sentry.capture_message("test", extra: { debug: true })
=> nil
```

So I must be doing something wrong with that means of testing, and it's not necessarily a confirmation of what's wrong with the rest of the config ☹️

**Comment by souredoutlook** [comment]
Hi @jywarren can you try using a DSN with the format `https://<public_key>@sentry.io/<project_id>` instead of the oXXXXXX.ingest.sentry.io subdomain? 

It's possible to get verbose logging by configuring the logger option, you can read more about it here: https://docs.sentry.io/platforms/ruby/configuration/options/ 

I believe you'll need to set the logging level in the app to be verbose as Sentry will respect logging levels.
